src/any_llm/utils/reasoning.py

```
@@ -25,14 +25,21 @@ def find_reasoning_tag(text: str, opening: bool = True) -> tuple[int, str] | Non
     return (earliest_pos, earliest_tag) if earliest_tag else None
 
 
-def is_partial_reasoning_tag(text: str, opening: bool = True) -> bool:
-    """Check if text could be the start of any reasoning tag."""
+def partial_reasoning_tag_suffix_len(text: str, opening: bool = True) -> int:
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)